circle.yml

```diff
@@ -4,6 +4,7 @@ machine:
     GIT_SOURCE_REPO: https://github.com/git/git.git
     GIT_EARLIEST_SUPPORTED_VERSION: v2.0.0
     GIT_LATEST_SOURCE_BRANCH: master
+    GIT_NEXT_SOURCE_BRANCH: next
```

Comment:
Is this the same line as https://github.com/git-lfs/git-lfs/pull/2470#discussion_r130989750?